hepcrawl/parsers/jats.py

```diff
@@ -502,11 +502,11 @@ def get_reference(self, ref_node):
 
         citation_node = ref_node.xpath('./mixed-citation')
 
-        raw_ref = remove_tags(
-            citation_node[0],
-            strip='self::object-id | self::*[@specific-use="suppress-display"]'
-        ).strip()
-        builder.add_raw_reference(raw_ref, source=self.builder.source)
+        builder.add_raw_reference(
+            etree.tostring(ref_node.root).strip(),
```

Comment:
you shouldn't ever have to use the `etree` API: `ref_node.extract().strip()` instead.